package.json

```diff
@@ -21,6 +21,13 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git://github.com/christopherbradleybanks/eslint-plugin-folders.git"
+    "url": "git+ssh://git@github.com/pajarraco/eslint-plugin-folders.git"
+  },
+  "bugs": {
+    "url": "https://github.com/pajarraco/eslint-plugin-folders/issues"
+  },
+  "homepage": "https://github.com/pajarraco/eslint-plugin-folders#readme",
+  "directories": {
+    "lib": "lib"
```

Comment:
this should not be included in the change

Reply:
I changed back